seatunnel-transforms-v2/src/main/java/org/apache/seatunnel/transform/sql/zeta/functions/udf/ZetaUDF.java

```diff
@@ -15,7 +15,7 @@
  * limitations under the License.
  */
 
-package org.apache.seatunnel.transform.sql.zeta;
+package org.apache.seatunnel.transform.sql.zeta.functions.udf;
```

Comment:
why move the package of ZetaUDF?

Reply:
This will cause incompatibility with older versions